tests/test_test_outcome.m

```diff
@@ -84,5 +98,6 @@ function helper_test_outcome(class_,args,...
 function s=rand_str()
     s=char(20*rand(1,10)+65);
 
-
+function assert_contains(a,b)
```

Comment:
Why not call this `assertContains`, for consistency with the outward facing `assert*` functions?